README.md

```diff
@@ -34,64 +38,34 @@ yarn test_sband <options>
 
+Step 5: Set up the AlbertaSat GNURadio Ground Station, which uses [radioconda](https://github.com/ryanvolz/radioconda). This will install our environment and all blocks needed, so there is no need to manually install other dependencies.
 ```
-conda create -n gnuradio-3.9.4
-conda activate gnuradio-3.9.4
-conda config --env --add channels conda-forge
-conda config --env --set channel_priority strict
-conda install gnuradio python=3.9.4
-sudo apt install liblog4cpp5v5 liblog4cpp5-dev
-conda install -c conda-forge gnuradio-satellites
-sudo apt install libuhd-dev uhd-host
-git clone https://github.com/AlbertaSat/ex2_sdr
+conda create -n radioconda -c conda-forge -c ryanvolz --only-deps radioconda
+conda activate radioconda
+conda install --file https://github.com/ryanvolz/radioconda/releases/download/2023.07.26/radioconda-win-64.lock
+gnuradio-companion
 ```
 
-Step 3: Install dependencies and build the ground station.
-
-```
-yarn install_dependencies
-yarn csp:clone
-yarn uhf:clone
-yarn build:gnuradio
-```
+This should open GNURadio 3.10.7. Open and run the relevant flow graph for your application in `libcsp/ex2_sdr/gnuradio/uhf/` or  `libcsp/ex2_sdr/gnuradio/sband/`. Note: the first time this is run on a new machine, gnuradio will prompt you to run uhd_images_downloader.py from the proper directory path.  
 
-Step 4: Run the ground station. This must be done every new session.
-```
-conda activate gnuradio-3.9.4
-gnuradio-companion
-```
-In GNURadio, open and run the relevant flow graph for your application in `ex2_sdr/gnuradio/uhf/` or  `ex2_sdr/gnuradio/sband/`. Note: the first time this is run on a new machine, gnuradio will prompt you to run uhd_images_downloader.py from the proper directory path. Do that, and then try running the flow graph again. Next, you can run the relevant ground station application (e.g. cli, sat_cli, ftp) using yarn, making sure to use the correct arguments for what you're doing.
-```
-yarn cli -I sdr -u
-```
-Step 5: Gpredict install. This is only necessary when tracking a satellite that is in orbit.
+Step 6: Install Gpredict on your machine, if it's not already present. This is only necessary when tracking a satellite that is in orbit. If you want to run the ground station without doppler correction, you will likely need to disable the Gpredic doppler
 
 ```
 sudo apt install gpredict
 gpredict
 ```
+
 In Gpredict preferences, set your ground station location. Also, set TLE auto-update to daily, and have "If TLEs are too old:" set to "Perform automatic update in the background." Create a new module in the "File" tab with the satellites of interest selected.
 
-Step 6: Doppler shift correction install. This is only necessary when tracking a satellite that is in orbit. Ensure gnuradio-companion is closed before proceeding.
```

Comment:
Why is this doppler stuff irrelevant?

Reply:
Because it's included in radioconda already! So the block is supported in gnuradio